erpnext/bulk_transaction/doctype/bulk_transaction_log/bulk_transaction_log.js

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2021, Frappe Technologies Pvt. Ltd. and contributors
+// For license information, please see license.txt
+
+frappe.ui.form.on('Bulk Transaction Log', {
+
+	refresh: function(frm) {
+		frm.add_custom_button(__('Retry Transactions'), ()=>{
```

Comment:
- This shouldn't be shown if there's nothing failed. 
- The button is present on document indicating the whole document will be retried, though it's hardcoded to only retry today's failures. Very bad UX IMO. 

<img width="1112" alt="Screenshot 2022-02-02 at 5 51 50 PM" src="https://user-images.githubusercontent.com/9079960/152152960-70dec54b-3a75-4b34-9e52-16128e70be2f.png">